internal/namespaces/k8s/v1beta4/helpers_test.go

```diff
@@ -0,0 +1,122 @@
+package k8s
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+	k8s "github.com/scaleway/scaleway-sdk-go/api/k8s/v1beta4"
+)
+
+func init() {
+	if !core.UpdateCassettes {
+		k8s.RetryInterval = 0
+	}
+}
+
+const (
+	kapsuleVersion = "1.17.3"
```

Comment:
Do we need to update this value regulary? What we happen when this version is not supported?

Reply:
Just need to update it while updating the cassettes (if the version is not supported anymore) so it's safe since it will fail when updating locally